user-registration-app/src/app/views/live-view/live-view.component.html

```diff
@@ -305,279 +303,427 @@
+              </div>
+              <div class="card-content pt-5">
+                <span class="card-title activator text-darken-4" style="font-weight:bold;">
+                  Introduction to Machine Learning</span>
+                <div style="font-weight:bold;">Neil Ashtekar, Nittany Data Labs</div>
+                <div style="font-weight:bold;">Location: Auditorium Business Building (Workshop Room 4)</div>
                 <div style="font-weight:bold;">Time: 4:00 PM - 4:45 PM Saturday</div>
               </div>
               <div class="card-reveal">
-                <span class="card-title">Designing a Great User Experience<i
+                <span class="card-title">Introduction to Machine Learning<i
                   class="material-icons right">close</i></span>
-                <p>In this workshop, you will learn user-centered design strategies
-                  for creating a product that is intuitive and easy to use. You will
-                  come away with an understanding of established design practices as
-                  well as some tips and tricks to help guide the design process!
+                <p> Interested in machine learning? Want to understand how algorithms can learn from and
+                  make predictions on data? Come to the Machine Learning Workshop here at HackPSU! We will
+                  see how these algorithms work and show you how to create neural networks, decision trees,
+                  and more in only a few lines of code.
                 </p>
+                <p class="card-requirements">This workshop has software requirements:</p>
+                <ul>
+                  <li class="wrksp-req">Anaconda</li>
+                </ul>
               </div>
             </div>
             <div class="card intermediate">
-              <div class="card-image waves-effect waves-block waves-light">
-                <img class="activator" src="../../../assets/images/workshop_leaders/hackpsu.png">
-                <a class="btn halfway-fab waves-effect waves-light red activator">click to learn more</a>
+              <div class="card-image">
+                <img class="activator" src="../../../assets/images/workshop_leaders/EthanHall.HEIC">
+                <a class="waves-effect waves-light activator red btn">click to learn more</a>
               </div>
               <div class="card-content pt-5">
-                <span class="card-title activator" style="font-weight:bold;">Mock Technical Interviews</span>
-                <div class="text-darken-4" style="font-weight:bold;">HackPSU Technology Team and Friends!</div>
-                <div style="font-weight:bold;">Location: Auditorium (Workshop Room 4)</div>
-                <div style="font-weight:bold;">Time: 6:00 PM - 6:45 PM Saturday</div>
+                <span class="card-title activator text-darken-4" style="font-weight:bold;">
+                  Designing a Great User Experience</span>
```

Comment:
The title doesn't seem to match with the revealed title

Reply:
This has been fixed